core/src/main/java/org/fao/geonet/kernel/Thesaurus.java

```diff
@@ -965,13 +983,20 @@ private void retrieveThesaurusInformation(Path thesaurusFile, String defaultTitl
                 this.defaultNamespace = DEFAULT_THESAURUS_NAMESPACE;
             }
 
-            Element dateEl = Xml.selectElement(thesaurusEl, "skos:ConceptScheme/dcterms:issued|skos:Collection/dc:date", theNSs);
+            Element issuedDateEl = Xml.selectElement(thesaurusEl, "skos:ConceptScheme/dcterms:issued|skos:Collection/dc:date", theNSs);
```

Comment:
I think issuedDateEl should only be `skos:ConceptScheme/dcterms:issued`

dateEl should be the old value `skos:ConceptScheme/dcterms:issued|skos:Collection/dc:date` or it should be set to issuedDateEl  and if null then attempt to use `skos:Collection/dc:date`

Reply:
i had reverted the default date code to exactly original. So the only change are the newly added created, issued, modified date

Reply:
You still did not remove dc:date from issuedDateEI

Reply:
removed